core-bundle/src/Resources/contao/library/Contao/Model.php

```diff
@@ -1082,13 +1082,9 @@ protected static function find(array $arrOptions)
 			$objStatement->limit($arrOptions['limit'], $arrOptions['offset']);
 		}
 
-		if (!\array_key_exists('value', $arrOptions))
-		{
-			$arrOptions['value'] = array();
-		}
-
 		$objStatement = static::preFind($objStatement);
-		$objResult = $objStatement->execute(...(\is_array($arrOptions['value']) ? $arrOptions['value'] : array($arrOptions['value'])));
+
+		$objResult = $objStatement->execute(...((array) ($arrOptions['value'] ?? array())));
```

Comment:
```php
if (!isset($arrOptions['value']))
{
	$arrOptions['value'] = array();
}

$objStatement = static::preFind($objStatement);
$objResult = $objStatement->execute(...((array) $arrOptions['value']));
```

Reply:
Changed in 75031d349c9f2bd780bf15999599684be1bb9d04. This is CS right? Because it's IMHO functionally equivalent.

Reply:
Yes, see my comment above.

> It is the same as your variant, of course, but maybe less complex?